docker/dockerfiles/Dockerfile.dev

```diff
@@ -4,8 +4,8 @@ FROM node:${NODE_VERSION}-bookworm-slim
 
 ENV PYTHONUNBUFFERED=1
 
-ENV DEBUG=True
-ENV ENVIRONMENT=development
+ENV DEBUG ${DEBUG:-True}
+ENV ENVIRONMENT ${ENVIRONMENT:-development}
```

Comment:
Just for the sake of consistency, can we keep the `ENV VARIABLE=VALUE` (i.e., with an equals sign for assignment)? 
Or is there a reason we can't keep it in this way?